tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -189,6 +189,12 @@ HDMI_IN_CAPTURE=true"
 NHLT_BIN=nhlt-sof-mtl-hdmi-ssp02.bin,HEADSET_CODEC=false,HDMI_IN_CAPTURE=true,\
 HDMI_IN_1_ID=0,HDMI_IN_2_ID=1"
 
+#ES83x6 codec alone without HDMI-in capture
+"cavs-es83x6\;sof-mtl-es83x6-ssp1\;PLATFORM=mtl,PREPROCESS_PLUGINS=nhlt,\
+NHLT_BIN=nhlt-sof-mtl-es83x6-ssp1.bin,HEADSET_SSP_DAI_INDEX=1,\
+HEADSET_CODEC=true,HEADSET_CODEC_NAME=SSP1-Codec,HDMI1_ID=3,HDMI2_ID=4,HDMI3_ID=5,\
+HDMI_IN_CAPTURE=false,USE_CHAIN_DMA=true"
```

Comment:
@ujfalusi @bardliao do we intend to use chain DMA for SSP  or should this use DSP ?

Reply:
Chain DMA is only available for HD-DMA endpoints, SSP in MTL is GPDMA serviced.

Reply:
@lgirdwood : no impact for SSP use case. hence removed it. please check now.

Reply:
fyi: ChainDMA is enabled by default for HDMI PCMs

Reply:
> fyi: ChainDMA is enabled by default for HDMI PCMs

=> Thanks.